app/Http/Controllers/AuthController.php

```diff
@@ -83,45 +75,38 @@ public function refreshToken(RefreshTokenRequest $request, JWTAuth $auth): JsonR
             $newToken = $auth->fromUser($user);
             $tokenCookie = $this->makeAuthorizationTokenCookie($newToken, $remember);
 
-            return response()
-                ->json([
-                    'token' => $newToken,
-                    'ttl' => config('jwt.ttl'),
-                    'refresh_ttl' => config('jwt.refresh_ttl')
-                ])
-                ->withHeaders(['Authorization' => "Bearer {$newToken}"])
-                ->withCookie($tokenCookie);
+            return RefreshTokenResource::make([], $newToken, $tokenCookie);
         } catch (JWTException $e) {
             throw new UnauthorizedHttpException('jwt-auth', $e->getMessage(), $e, $e->getCode());
         }
     }
 
-    public function logout(LogoutRequest $request): Response
+    public function logout(LogoutRequest $request): LogoutResource
     {
         $tokenCookie = $this->makeAuthorizationTokenExpiredCookie();
 
-        return response('', Response::HTTP_NO_CONTENT)->withCookie($tokenCookie);
+        return LogoutResource::make([], $tokenCookie);
     }
 
-    public function forgotPassword(ForgotPasswordRequest $request, UserService $service): Response
+    public function forgotPassword(ForgotPasswordRequest $request, UserService $service): ForgotPasswordResource
     {
         $service->forgotPassword($request->input('email'));
 
-        return response('', Response::HTTP_NO_CONTENT);
+        return ForgotPasswordResource::make([]);
```

Comment:
I not sure that we need to use resources for the no content response, it looks we should return response directly in such cases